lib/community/utils/post_card_action_helpers.dart

```diff
@@ -243,6 +244,15 @@ final List<ExtendedPostCardActions> postCardActionItems = [
     getOverrideIcon: (postView) => postView.read ? Icons.mark_email_unread_rounded : Icons.mark_email_read_outlined,
     shouldEnable: (isUserLoggedIn) => isUserLoggedIn,
   ),
+  ExtendedPostCardActions(
+    postCardAction: PostCardAction.hide,
+    label: l10n.hide,
+    getOverrideLabel: (context, postView) => postView.hidden == true ? l10n.unhide : l10n.hide,
+    icon: Icons.close_rounded,
```

Comment:
Thoughts about using `visibility` and `visibility_off` for the icons?

Reply:
Ah yes those match the Lemmy UI. Perfect. Should be good to go now!